File: Common/InMotionGIT.Common.Domain/DataType/Result.cs
using System;
using System.ComponentModel;
using System.Runtime.Serialization;
using System.Xml.Serialization;

namespace InMotionGIT.Common.Domain.DataType;


[DataContract()]
[Serializable()]
public class Result
{

    [DataMember(EmitDefaultValue = false)]
    [XmlAttribute()]
    [DefaultValue(typeof(bool), "False")]
    public bool Success { get; set; }

    [DataMember(EmitDefaultValue = false)]
    [XmlAttribute()]
    [DefaultValue(typeof(int), "0")]
    public int Code { get; set; }

    [DataMember(EmitDefaultValue = false)]
    [XmlAttribute()]
    [DefaultValue(typeof(string), "")]
    public string Reason { get; set; }

}

File: Common/InMotionGIT.Common/DataType/Result.vb
Imports System.ComponentModel
Imports System.Runtime.Serialization
Imports System.Xml.Serialization

Namespace DataType

    <DataContract()>
    <Serializable()>
    Public Class Result

        <DataMember(EmitDefaultValue:=False)>
        <XmlAttribute(), DefaultValue(GetType(Boolean), "False")>
        Public Property Success As Boolean

        <DataMember(EmitDefaultValue:=False)>
        <XmlAttribute(), DefaultValue(GetType(Integer), "0")>
        Public Property Code As Integer

        <DataMember(EmitDefaultValue:=False)>
        <XmlAttribute(), DefaultValue(GetType(String), "")>
        Public Property Reason As String

    End Class

End Namespace